.claude/commands/shared/task-management-patterns.yml

```
@@ -0,0 +1,287 @@
+    Partial_Import: "Import recoverable data, start fresh for rest"
+    Checkpoint_Rollback: "Restore from last known good state"
+
+Task_Validation:
+  Integrity_Checks: "Verify task structure & required fields"
+  Context_Completeness: "Ensure critical context elements present"
+  Progress_Consistency: "Validate todo progress vs task progress"
+  Git_Alignment: "Verify git state matches task expectations"
+```
+
+---
+*Task Management Patterns v3 - Consolidated task tracking, todo integration, and workflow management*
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)